build.sh

```diff
@@ -0,0 +1,40 @@
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
For context, the changes on this branch are meant to be for the general public and should not include code or configurations specific to an internal setup or internal development. 

If, for whatever reason, the example needs a particular configuration, it should be done via CMake rather than `.sh` files